antora/docs/modules/ROOT/templates/policy_configuration.hbs

```diff
@@ -212,3 +212,65 @@ interchangeably with the `exclude` field.
 
 Please make the required tooling changes soon to prevent breakages when the
 `non_blocking_checks` field is removed in the future.
+
+== Data Sources
+
+Some of the Enterprise Contract policy rules, defined in the ec-policies git
+repository, rely on certain data values when evaluated. For example, a policy
+rule exists to ensure all the parent container images used during the build
+process have been retrieved from an acceptable registry. The list of acceptable
+registries is a data value. This type of data is called Rule Data.
+
+In addition to policy rules, the ec-policies git repository also defines values
+for Rule Data, see
+link:https://github.com/hacbs-contract/ec-policies/blob/main/data/rule_data.yml[data/rule_data.yml]
```

Comment:
Was the intent of including the symlink to `rule_data.yml` to link to it from here relatively? I think what's needed here is to use [attachments](https://docs.antora.org/antora/latest/page/attachments/).